t/ui/15-search.t

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env perl
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use Test::Most;
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use Test::Mojo;
+use Test::Warnings ':report_warnings';
+
+use OpenQA::Test::Case;
+use OpenQA::Client;
+
+use OpenQA::SeleniumTest;
+
+OpenQA::Test::Case->new->init_data(skip_fixtures => 1);
+
+my $driver = call_driver();
+if (!$driver) {
+    plan skip_all => $OpenQA::SeleniumTest::drivermissing;
+    exit(0);
+}
+
+my $t = Test::Mojo->new('OpenQA::WebAPI');
+# we need to talk to the phantom instance or else we're using wrong database
+my $url = 'http://localhost:' . OpenQA::SeleniumTest::get_mojoport;
+
+subtest 'Perl modules' => sub {
+    my $search = $driver->find_element_by_id('global-search');
+    $search->click();
+    is($search->get_text(), '', 'Empty search entry by default');
+
+    $search->send_keys('timezone');
+    $search->send_keys(Selenium::Remote::WDKeys->KEYS->{'enter'});
+    wait_for_ajax;
+
+    like($driver->get_title(), qr/Search/, 'Search shown') or return;
+    my $header = $driver->find_element_by_id('results-heading');
+    is($header->get_text(), '2 matches found', 'Number of results in header');
+    my $results = $driver->find_element_by_id('results');
+    my @entries = $results->children('.list-group-item');
+    is(scalar @entries, 2, 'two elements') or return;
+
+    my $first = $entries[0];
+    is(
+        $first->child('#occurrence')->get_text(),
+        'opensuse/tests/installation/installer_timezone.pm',
+        'Expected occurrence'
+    );
+    is($first->child('#contents')->get_text(), '', 'No contents');
+
+    my $second = $entries[1];
+    is(
+        $second->child('#occurrence')->get_text(),
+        'opensuse/tests/installation/installer_timezone.pm',
+        'Expected occurrence'
+    );
+    is(
+        $second->child('#contents')->get_text(),
+        qq{    assert_screen "inst-timezone", 125 || die 'no timezone';},
+        'Expected contents'
+    );
+};
+
+kill_driver();
+
+done_testing();
```

Comment:
please check if the background chromedriver is properly cleaned up if a test fails. I think this should look like in https://github.com/os-autoinst/openQA/blob/a684f1ef2e5d9f5c35ae66e3af74544ae346dd75/t/full-stack.t#L381 :

```perl
END { kill_driver };
done_testing;
```